src/odfesqlodbc/es_statement.cpp

```diff
@@ -148,6 +151,36 @@ RETCODE ExecuteStatement(StatementClass *stmt, BOOL commit) {
     return CleanUp();
 }
 
+SQLRETURN GetNextResultSet(StatementClass *stmt) {
+    if (stmt == NULL) {
```

Comment:
Do you need this check here, if SC_get_Result could handle input NULL and return NULL then you would only need a single check.